content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -304,6 +304,48 @@ properties:
     kind: Bar
 ```
 
+#### `olm.deprecations`
+
+Operator authors can use the file-based catalog (FBC) deprecation schema to provide relevant information for support and upgrades for users running operators. The deprecation schema follows the Meta FBC Schema conventions. 
+
+Deprecation for packages/bundles/channels is enabled through the olm.deprecations schema. This schema consists of a list of scoped references to catalog content, with a descriptive message string that provides more information to the user. The deprecation schema is an optional feature.
+
+A valid depreciation schema meets the following criteria:
+- There must be only one schema per package
+- The message must be a non-zero length
+- The package name must exist in the bundle
+  
+The deprecation feature does not consider overlapping deprecation (package vs channel vs bundle).
+
+|              | `olm.package`                   | `olm.channel`        | `olm.bundle`          |
+|--------------|---------------------------------|----------------------|-----------------------|
+| Scope        | Entire Package                  | Single Channel       | Single Bundle Version |
+| Requirements | Must not be an associate `name` | `name`  is mandatory | `name`  is mandatory  |
+
+> **Warning:** If the deprecation schema is invalid the entire FBC is deemed invalid by `bundle validate`
+
+Here is an example demonstrating each of the deprecation entry types:  
+```yaml
+schema: olm.deprecations
+package: kiali
+entries:
+  - reference:
+  	schema: olm.bundle
+  	name: kiali-operator.v1.68.0
+    message: |
+   	kiali-operator.v1.68.0 is deprecated. Uninstall and install kiali-operator.v1.72.0 for support.
+  - reference:
+  	schema: olm.package
+    message: |
+   	package kiali is end of life.  Please use 'kiali-new' package for support.
+  - reference:
+  	schema: olm.channel
+  	name: alpha
+    message: |
+   	channel alpha is no longer supported.  Please switch to channel 'stable'.
+```
```

Comment:
While I know this is an example that has been used in the past, I have a feeling that in our official documentation we should avoid using the name of a real operator _or_ include a disclaimer that explicitly states this is a documentation example only and that we are in no way stating that the deprecations here are from the kiali maintainers. I mainly just want to avoid this showing up in any google indexing if someone searches "kiali deprecated" and jumping to conclusions.

I personally would lean towards a change like:
```suggestion
```yaml
schema: olm.deprecations
package: deprecation-example
entries:
  - reference:
  	schema: olm.bundle
  	name: deprecation-example-operator.v1.68.0
    message: |
   	deprecation-example-operator.v1.68.0 is deprecated. Uninstall and install deprecation-example-operator.v1.72.0 for support.
  - reference:
  	schema: olm.package
    message: |
   	package deprecation-example is end of life.  Please use 'non-deprecated-example' package for support.
  - reference:
  	schema: olm.channel
  	name: alpha
    message: |
   	channel alpha is no longer supported.  Please switch to channel 'stable'.
```
```